spec/functional/resource/group_spec.rb

```diff
@@ -58,6 +58,7 @@ def group_should_not_exist(group)
     when "debian", "fedora", "rhel", "suse", "gentoo", "slackware", "arch"
       expect { Etc.getgrnam(group) }.to raise_error(ArgumentError, "can't find group for #{group}")
     when "windows"
+      sleep(5)
```

Comment:
if this works it would be preferable if we could add a check of some kind for readiness rather than adding 5 seconds to the tests.